plugins/woocommerce/includes/admin/class-wc-admin-settings.php

```diff
@@ -380,15 +380,19 @@ class="<?php echo esc_attr( $value['class'] ); ?>colorpick"
 					// Textarea.
 					case 'textarea':
 						$option_value = $value['value'];
+						$show_desc_at_end = $value['desc_at_end'] ?? false;
```

Comment:
```suggestion
						$option_value     = $value['value'];
						$show_desc_at_end = $value['desc_at_end'] ?? false;
```

Make the code linter happy